go.mod

```diff
@@ -3,11 +3,14 @@ module github.com/openshift/client-go
 go 1.16
 
 require (
-	github.com/openshift/api v0.0.0-20220819133013-d8b9c3791a1a
+	github.com/openshift/api v0.0.0-20220822155657-561a81ffb263
 	github.com/spf13/pflag v1.0.5
 	golang.org/x/oauth2 v0.0.0-20220411215720-9780585627b5 // indirect
 	k8s.io/api v0.24.0
 	k8s.io/apimachinery v0.24.0
 	k8s.io/client-go v0.24.0
 	k8s.io/code-generator v0.24.0
+	sigs.k8s.io/structured-merge-diff/v4 v4.2.3
```

Comment:
Are we comfortable introducing these new deps for all client-go consumers? Are these required?

Reply:
> Are we comfortable introducing these new deps for all client-go consumers? Are these required?

This is required by apimachinery, so it's not novel.